docs/sources/dashboards/create-reports/index.md

```diff
@@ -107,10 +107,16 @@ You can include dynamic dashboards with panels or rows, set to repeat by a varia
 
 By default, reports use the saved time range of the dashboard. You can change the time range of the report by:
 
-- Saving a modified time range to the dashboard.
+- Saving a modified time range to the dashboard. Changing the dashboard time range without saving it doesn't change the time zone of the report.
 - Setting a time range via the **Time range** field in the report form. If specified, the custom time range overrides the time range from the report's dashboard.
 
-The page header of the report displays the time range for the dashboard's data queries. Dashboards set to use the browser's time zone use the time zone on the Grafana server.
+The page header of the report displays the time range for the dashboard's data queries.
+
+#### Report time zones
+
+Reports use the time zone of the dashboard from which they’re generated. 
```

Comment:
```suggestion
Reports use the time zone of the dashboard from which they’re generated. You can control the time zone for your reports by setting the dashboard to a specific time zone. Note that this affects the display of the dashboard for all users. 
```
I liked that from your previous suggestion, did you remove it on purpose?

Reply:
Probably not on purpose. It just got lost in all the comments